standalone/header-footer/webpack.config.js

```diff
@@ -1,9 +1,12 @@
+// Alias 'seek-style-guide' so 'seek-style-guide-webpack' works correctly
```

Comment:
I wish this wasn't necessary.  Is there a way around it by modifying `seek-style-guide-webpack`?

Reply:
Possibly, but I think this is actually easier to reason about anyway, even if it's a little hacky.